README.md

```diff
@@ -43,6 +43,24 @@ openshift.operations.project.names: ["default", "openshift", "openshift-infra"]
 The current default is "default", "openshift", "openshift-infra", "kube-system".
 The names must all be in lower-case to be properly matched.
 
+## Common Data Model
+If your OpenShift EFK cluster supports the common data model, and you want to
+use the common data model, there are some configuration parameters that you can
+set in `elasticsearch.yml`, under `openshift.config`:
+
+* `use_common_data_model` - boolean - default `false` - if true, tell the
+  plugin that you want to use the common data model
+* `project_index_prefix` - string - default `""` - set this to the string value
+  that project/namespace indices use as their prefix - for example, with the
+  common data model, if the namespace is `test`, the index name will be
+  `project.test.$uuid.YYYY.MM.DD`.  In this case, use `"project."` as the
```

Comment:
technically shouldn't they use "project" as the prefix? if they use "project." then our indices would end up as "project..$project.$uuid.$date"

I think we would want to discourage having a user provide the delimiter value in the case they don't provide it it could cause some grief